vars/terraform.groovy

```diff
@@ -116,25 +116,36 @@ def call(userConfig = [:]) {
               stage('💸 Report estimated costs') {
                 try {
                   Boolean commentReport = false
-                  // On Azure we can use the terraform plan to estimate the costs as it doesn't contains the most sensible secrets like AWS
-                  if (env.GIT_URL.contains('jenkins-infra/azure')) {
+                  Boolean commentComparison = false
+                  // On AWS we can use the terraform plan to estimate the costs as it doesn't contains most sensible secrets
+                  if (env.GIT_URL.contains('jenkins-infra/aws')) {
                     final String planFileUrl = "${env.BUILD_URL}artifact/terraform-plan-for-humans.txt"
                     sh "terraform show -json ${planFileUrl} > plan.json"
                     sh 'infracost breakdown --path plan.json --show-skipped --format json --out-file infracost.json'
+                    // Also try the experimental HCL method for comparison and upstream contrib
+                    sh 'infracost breakdown --path . --terraform-parce-hcl --show-skipped --format json --out-file infracost-hcl.json'
```

Comment:
```suggestion
                    sh 'infracost breakdown --path . --terraform-parse-hcl --show-skipped --format json --out-file infracost-hcl.json'
```
As per https://www.infracost.io/docs/guides/advanced_usage/#parsing-hcl-experimental.
Sounds like a typo: don't forget to update the azure command line as well ;)